tests/unittests/algo/pbt/test_pbt.py

```diff
@@ -549,6 +558,58 @@ def test_suggest_num_population_size_fork_completed(self, space: Space, mocker):
         pbt_sample_mock.assert_called_with(2)
         pbt_fork_mock.assert_called_with(2)
 
+    def test_suggest_copy_working_dir(self, space, mocker, tmp_path):
+        shutil.rmtree(tmp_path)
```

Comment:
I'm curious: Why do these tests have this `shutil.rmtree(tmp_path)` and `os.makedirs(tmp_path)`?
Shouldn't this tmp_path fixture give you an empty directory everytime?

Reply:
They should not indeed. I removed this tests since it is not relevant anymore. I'll check for the instance in functional tests.

Reply:
I confirm it was not necessary. The folder exists but it is guaranteed empty. I removed them in cd2ff722.